### ISSUE TITLE
Placeholder in text input field

### ISSUE THREAD

**Issue by thepranavjain** [Achiaga/carbon_footprint#25]
When text input field is clicked, the cursor is inside the input field. As it should be. But the placeholder text (e.g. "Audi A4") doesn't go away.

Also, should there be more generic placeholders like "Name of the vehicle", "electricity consumed in kWh", etc. ?

**Comment by Achiaga** [comment]
Yes. You are right about the placeholder. I will check it out now. Thanks.

I was thinking about putting an information button that opens a modal with a clear example and explanation.
